### PR TITLE
Regenerate internal code with latest code generator

### PR DESCRIPTION
**Pull request by jhendrixMSFT**
Fixed up generated const type names to ease transition to CGC.

### CODE REVIEW

File: sdk/data/azappconfig/autorest.md [hunk 1]
```diff
@@ -10,10 +10,58 @@ input-file:
...
+    };
+    $.head.parameters[7].items["x-ms-enum"] = {
+        "name": "SettingFields",
+        "modelAsString": true
+    };
+- from: swagger-document
+  where: $.paths./labels
+  transform: >
+    $.get.parameters[5].items["x-ms-enum"] = {
+        "name": "LabelField",
```
Comment (RickWinter) [inline sdk/data/azappconfig/autorest.md:49]:
```suggestion
        "name": "LabelFields",
```

File: sdk/data/azappconfig/autorest.md [hunk 2]
```diff
@@ -10,10 +10,58 @@ input-file:
...
+        "modelAsString": true
+    };
+    $.head.parameters[5].items["x-ms-enum"] = {
+        "name": "LabelField",
```
Comment (RickWinter) [inline sdk/data/azappconfig/autorest.md:53]:
```suggestion
        "name": "LabelFields",
```

**Review by RickWinter — APPROVED** [review]

File: sdk/data/azappconfig/autorest.md (hunk 2 above)
Comment (jhendrixMSFT) [reply sdk/data/azappconfig/autorest.md]: fixed

File: sdk/data/azappconfig/autorest.md (hunk 1 above)
Comment (jhendrixMSFT) [reply sdk/data/azappconfig/autorest.md]: fixed

**Pull request merged**